site/tests/integration/page-footer.test.ts

```diff
@@ -1,44 +1,48 @@
 import { expect, test } from "playwright-test-coverage";
 
-test("page footer navigation works", async ({ page }) => {
+test("page footer navigation works", async ({ page, browserName }) => {
+  test.skip(
+    browserName === "webkit",
+    "https://app.asana.com/0/1202542409311090/1202651551651725",
+  );
   await page.goto("/");
 
-  await page.locator("footer >> text=Block Hub").click();
-  await expect(page).toHaveURL("/hub");
-
-  //   uncomment when https://app.asana.com/0/1202542409311090/1202629493068301 is done
-  //   await page.locator("footer >> a:has-text('Documentation')").click();
-  //   await page.waitForURL("/docs");
-
-  //   await page.locator("footer >> a:has-text('Specification')").click();
-  //   await page.waitForURL("/docs/spec");
-
-  //   await page.locator("footer>> a:has-text('Publish a Block')").click();
-  //   await page.waitForURL("/docs/developing-blocks#publish");
-
-  await page.locator("footer >> a:has-text('Contact Us')").click();
-  await expect(page).toHaveURL("/contact");
+  await page.locator("footer").scrollIntoViewIfNeeded();
 
   const socialLinksSelector = page.locator(
     "[data-testid='footer-social-links']",
   );
-
-  await expect(socialLinksSelector.locator("a >> nth=0")).toHaveAttribute(
+  await expect(socialLinksSelector.locator("a").first()).toHaveAttribute(
     "href",
     "https://twitter.com/blockprotocol",
   );
-
-  await expect(socialLinksSelector.locator("a >> nth=1")).toHaveAttribute(
+  await expect(socialLinksSelector.locator("a").nth(1)).toHaveAttribute(
     "href",
     "/discord",
   );
-
-  await expect(socialLinksSelector.locator("a >> nth=2")).toHaveAttribute(
+  await expect(socialLinksSelector.locator("a").nth(2)).toHaveAttribute(
     "href",
     "https://github.com/blockprotocol/blockprotocol",
   );
-
   await expect(
     socialLinksSelector.locator("text=Star us on Github"),
   ).toHaveAttribute("href", "https://github.com/blockprotocol/blockprotocol");
+
+  await page.locator("footer >> a:has-text('Contact Us')").click();
+  await expect(page).toHaveURL("/contact");
```

Comment:
We don't have to click on every link if this slows down the test. It's possible to leverage await `expect(locator).toHaveAttribute("href", "/path/to/page")`. We jump between pages in header because it's a universal test – we also test routing with it.

This suggestion is optional.

Reply:
I think I'll do this then, since jumping between pages here, leads to timeout issues on safari

Reply:
Updated in [47592f2](https://github.com/blockprotocol/blockprotocol/pull/450/commits/47592f20cb448265c5baa3a94f80671da8609ae7)